### PR TITLE
chore: rename data-processing to pypi

### PR DESCRIPTION
**Pull request by bjwswang**
#### What type of PR is this?
/kind refactor

#### Which issue(s) this PR fixes

Fixes #

#### Special notes for your reviewer

Now we have more and more python projects which we should place them together.

- data_manipulation
- ragas_once https://github.com/kubeagi/arcadia/pull/513

### CODE REVIEW

**Comment by bjwswang** [comment]
@nkwangleiGIT  @ggservice007 @wangxinbiao   目前我们有两个python相关的项目:

- 数据处理
- ragas评估

为了管理方便，把两个项目都放到同一个目录 `pypi`下

**Comment by ggservice007** [comment]
@bjwswang  we can use the following structure.
pypi/data-processing
pypi/evaluations

If do as the above, no changes to code.

**Comment by bjwswang** [comment]
> @nkwangleiGIT @ggservice007 @wangxinbiao 目前我们有两个python相关的项目:
> 
> * 数据处理
> * ragas评估
> 
> 为了管理方便，把两个项目都放到同一个目录 `pypi`下

I remove the intermidiate directory `data-processing` and keeps `data_manipulation` . 
The original code moves under `pypi/data_manipulation/src`

File: pypi/data-processing/.gitignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# python
+__pycache__
+.ipynb_checkpoints
+
```
Comment (bjwswang) [inline pypi/data-processing/.gitignore:4]: @ggservice007 @wangxinbiao  is this correct?

**Comment by bjwswang** [comment]
@Lanture1064 any comments

**Review by ggservice007 — APPROVED** [review]

**Review by wangxinbiao — APPROVED** [review]

**Pull request merged**